lib/metadata/wrapper.js

```diff
@@ -117,9 +117,9 @@ const metadata = {
         });
     },
 
-    listObject: (bucketName, prefix, marker, delimiter, maxKeys, log, cb) => {
+    listObject: (bucketName, listingParams, log, cb) => {
```

Comment:
Performance-wise, they are not different because inside this function, we have to create an object out of the listing parameters anyway.